squema.sql

```diff
@@ -0,0 +1,12 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals
+(
+    id integer NOT NULL,
```

Comment:
- [OPTIONAL] It will be nice if the `id` gets incremented automatically. 

Example: ` id int NOT NULL IDENTITY(1,1)`